lib/spandex_phoenix/plug/start_trace.ex

```diff
@@ -31,8 +31,9 @@ defmodule SpandexPhoenix.Plug.StartTrace do
 
   @impl Plug
   def call(conn, opts) do
-    if opts[:filter_traces].(conn) do
-      begin_tracing(conn, opts)
+    case opts[:filter_traces].(conn) do
```

Comment:
Is there a reason to switch to `case` instead of just `if/else`? I normally do `if/else` when it's just a `case` with `true` and `false`.

Reply:
Didn't want to use `else` and wanted to differentiate between the `conn` returned by `begin_tracing` and the one received as an argument. Happy to change but I don't like the `if` construct that much 😅

Reply:
I generally would prefer if/else over case true/false, but not enough to reject your changes :)

Reply:
I have replaced this with a `if/else` statement. Realized it makes sense to keep code consistency.